web/public/themes/noir-gold.css

```
@@ -0,0 +1,566 @@
+  --green-dark: var(--success-500);
+  --green-bg:   var(--success-100);
+  --red:        var(--error-400);
+  --red-dark:   var(--error-500);
+  --red-bg:     var(--error-100);
+  --yellow:     var(--warning-400);
+  --yellow-dark:var(--warning-500);
+  --yellow-bg:  var(--warning-100);
+  --blue:       var(--gold-400);
+  --blue-bg:    var(--accent-bg);
+
+  --font-sans: "Source Serif 4", "IBM Plex Serif", Georgia, "Times New Roman", serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)